src/tags/object/PagedView.js

```diff
@@ -82,7 +82,8 @@ const HtxPagedView = observer(({ item }) => {
       if (obj.selected) {
         const _pageNumber = parseFloat(obj.object.name.split('_')[1]) + 1;
 
-        setPage(Math.ceil(_pageNumber / pageSize));
+        changePageSize(1);
```

Comment:
Is this needed if its always just 1 now?

Reply:
In this case it is change the page size to 1 to show just the item that user choose!